src/dbnode/storage/shard.go

```diff
@@ -2676,7 +2676,9 @@ func (s *dbShard) AggregateTiles(
 	openBlockReaders := make([]fs.DataFileSetReader, 0, len(blockReaders))
 	defer func() {
 		for _, reader := range openBlockReaders {
-			_ = reader.Close()
+			if err := reader.Close(); err != nil {
+				s.logger.Error("error closing DataFileSetReader", zap.Error(err))
```

Comment:
nit: typically no `error` in go error returns, change to something like "could not close DataFileSetReader"

Reply:
👍